frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -118,10 +109,39 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
     const usableConfigs = globalConfigs
       .filter((item) => this.checkFlags(item))
       .map((item) => item.properties);
-    const allItems = usableConfigs.length > 0 && items.concat(usableConfigs);
-    const sortedItems =
-      usableConfigs.length > 0 ? _.sortBy(_.flatten(allItems), 'kind', 'asc') : items;
-
+    let allItems = items.concat(usableConfigs);
+    allItems = allItems
+      .map((item) => {
```

Comment:
We're already mapping in `componentDidMount`. We shouldn't transform it twice. move this logic there.

Reply:
Hi, moved all to `componentDidMount`, thanks